src/Symfony/Component/Security/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+ * Deprecated `IS_AUTHENTICATED_ANONYMOUSLY` in favor of `IS_AUTHENTICATED`
```

Comment:
This does not make sense to me to be honest. In relation to your comment in the PR;

> `IS_AUTHENTICATED_ANONYMOUSLY` actually meant "is authenticated, either anonymous or fully". As this is confusing, it is replaced by `IS_AUTHENTICATED`

So we would have `IS_AUTHENTICATED` and `IS_AUTHENTICATED_FULLY`, I'm pretty sure this will cause more confusion than it solves. The `IS_` function define a minimum authentication level, which also sounds more like it _should_ be connected to the firewall than just access control. You can't have `anonymous: ~` on your firewall and leave out the `IS_AUTHENTICATED` (or `IS_AUTHENTICATED_ANONYMOUSLY`). When you combine this, the name makes no more sense at all.